pyscroll/data.py

```diff
@@ -43,8 +44,10 @@ class PyscrollDataAdapter:
     # or properties.  they are listed here as class
     # instances, but use as properties is fine, too.
 
-    tile_size = None  # (int, int): size of each tile in pixels
-    map_size = None  # (int, int): size of map in tiles
+    # (int, int): size of each tile in pixels
+    tile_size: tuple[int, int] = (0, 0)
+    # (int, int): size of map in tiles
+    map_size: tuple[int, int] = (0, 0)
```

Comment:
tile_size and map_size were the cause of a lot of typehints, alternatively we can use a lot of asserts

Reply:
thats fine